src/elements/content-sidebar/activity-feed/task-new/PendingAssignment.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import Button from '../../../../components/button';
+import messages from '../../../common/messages';
+import { ACTIVITY_TARGETS } from '../../../common/interactionTargets';
+import { TASK_TYPE_APPROVAL } from '../../../../constants';
+
+type Props = {
+    onTaskApproval: Function,
+    onTaskComplete: Function,
+    onTaskReject: Function,
+    taskType?: TaskType,
+};
+
+const PendingAssignment = ({ onTaskApproval, onTaskReject, onTaskComplete, taskType }: Props): React.Node => (
+    <div className="bcs-task-pending-assignment bcs-task-assignment-actions">
+        {taskType === TASK_TYPE_APPROVAL ? (
+            <React.Fragment>
+                <Button
+                    className="bcs-task-action-button bcs-task-check-btn"
+                    onClick={onTaskApproval}
+                    data-resin-target={ACTIVITY_TARGETS.TASK_APPROVE}
+                >
```

Comment:
Please add `type="button"`, as these are submit buttons by default.

Reply:
The Form component relies on intercepting submit events, so this is right